src/System.Management.Automation/engine/Utils.cs

```diff
@@ -478,13 +478,22 @@ internal static string GetWindowsPowerShellVersionFromRegistry()
         }
 #endif
 
-        internal static string DefaultPowerShellAppBase => GetApplicationBase(DefaultPowerShellShellID);
```

Comment:
I guess we can remove all the reference to `DefaultPowerShellAppBase` now, and just `GetApplicationBase()` directly instead.